### PR TITLE
Fix final nits

### PR DESCRIPTION
**Pull request by naijwu**
## Description

- changed theme colours
- removed nwhacks2021 decals in Landing component
- textbox resize fix and required markers added for resume + link text inputs under `Skills.js`

notes:
- flags for redirect to submission page is controlled under `InternalWebsites` > `Livesite` > `submissionsOpen` in nwplus-ubc firestore
- Continue application button disable state is also controlled under Livesite doc as `applicationsOpen`

#### Workaround used:
Code for empty github or portfolio link (depending on `contributionRole` as `"developer"` or `"designer"` respectively) is a work-around... doesn't have the exact same behavior as the longAnswers (after submission, when making the link empty again, the error doesn't pop up, contra other fields where where the empty textfield check occurs on every onChange)

### CODE REVIEW

File: src/components/ApplicationForm/Skills.js [hunk 1]
```diff
@@ -115,9 +138,8 @@ export default ({ refs, errors, formInputs, onChange, role, handleResume }) => {
 
           {role === 'designer' ? (
             <>
-              <FormRow fieldValue="Personal website/portfolio link">
+              <FormRow fieldValue="Personal website/portfolio link" required>
                 <TextInput
-                  inline
```
Comment (anneguo3) [inline src/components/ApplicationForm/Skills.js:120]: There's a funky bug - whereby if I type a space into any of these fields it seems to crash the website and become unresponsive 😭, but there isn't a clear log of an error stack either? The error doesn't exist on the long answer question's boxes though

**Comment by anneguo3** [comment]
Landing screen on mobile doesn't handle the nwHacks logo 👀 (unable to login) 
Screen Shot 2021-12-01 at 9 29 40 AM

Can the text for the sub-questions stay the same white text as the initial question?
Screen Shot 2021-12-01 at 9 43 27 AM

There's also some weird behaviour with our input handlers taking longer than expected to respond - pinged @ianmah about it but unsure if this is something we need to look into/triage before release - all the other changes look really good though!! 🎉

**Review by kevin-zou — APPROVED** [review]
Banger PR 👏 - everything looks great and should be good to merge after you address Anne's comment (I think you can fix the first one by changing styling for `StyledLogoLockup` in `Landing/index.js`)

**Review by anneguo3 — APPROVED** [review]
looks amazing - thank you for those fixes!! 🎉

File: src/components/ApplicationForm/Skills.js [hunk 2]
```diff
@@ -148,9 +169,8 @@ export default ({ refs, errors, formInputs, onChange, role, handleResume }) => {
             </>
           ) : (
             <>
-              <FormRow fieldValue="GitHub/BitBucket/GitLab">
+              <FormRow fieldValue="GitHub/BitBucket/GitLab" required>
```
[the remark this answers is not in the document]
Comment (naijwu) [reply src/components/ApplicationForm/Skills.js:176]: WILL GET ON THIS

**Review by kevin-zou — APPROVED** [review]
LFG 🔥 looks good to me!

**Pull request merged**